specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -5251,15 +5306,38 @@
           ],
```

Comment:
These are not the canonical values for provisioningStatus.
They must be 
'Succeeded'
'Failed'
'Canceled'
plus whatever other NON-TERMINAL states you want.

'Completed' sounds like its an improper terminal state, unfortunately.

[https://github.com/Azure/azure-resource-manager-rpc/blob/c32a616596b9547d391abdd6a45b6a85e6b3936b/v1.0/async-api-reference.md#provisioningstate-property](https://github.com/Azure/azure-resource-manager-rpc/blob/c32a616596b9547d391abdd6a45b6a85e6b3936b/v1.0/async-api-reference.md#provisioningstate-property)